test/unit/app/presenters/publishing_api/editionable_worldwide_organisation_presenter_test.rb

```diff
@@ -0,0 +1,38 @@
+require "test_helper"
+
+class PublishingApi::EditionableWorldwideOrganisationPresenterTest < ActiveSupport::TestCase
+  def present(...)
+    PublishingApi::EditionableWorldwideOrganisationPresenter.new(...)
+  end
+
+  test "presents a Worldwide Organisation ready for adding to the publishing API" do
+    worldwide_org = create(:editionable_worldwide_organisation)
+
+    public_path = worldwide_org.public_path
+
+    expected_hash = {
+      base_path: public_path,
+      title: worldwide_org.title,
+      schema_name: "worldwide_organisation",
+      document_type: "editionable_worldwide_organisation",
+      locale: "en",
+      publishing_app: Whitehall::PublishingApp::WHITEHALL,
+      rendering_app: Whitehall::RenderingApp::GOVERNMENT_FRONTEND,
+      public_updated_at: worldwide_org.updated_at,
+      routes: [{ path: public_path, type: "exact" }],
+      redirects: [],
+      details: {
+        logo: {
+          crest: "single-identity",
+        },
+      },
+      update_type: "major",
+    }
+
+    presented_item = present(worldwide_org)
+
+    assert_equal expected_hash, presented_item.content
+    assert_equal "major", presented_item.update_type
+    assert_equal worldwide_org.content_id, presented_item.content_id
+  end
```

Comment:
Would it be possible to add an assertion that the presented item is valid against the schema, e.g. https://github.com/alphagov/whitehall/blob/b814a4ebd2600285ddd226ca5a135e156638199a/test/unit/app/presenters/publishing_api/news_article_presenter_test.rb#L196-L197

Reply:
Added with a TODO note as the assertion currently fails, and wont pass till more work is done on the model

Reply:
Could you write Trello cards for the things that will need to be added to this presenter in order to make this test pass (I think it's just `formatted_title` for now).

Also, it should be safe to add a test for the links now:

```ruby
assert_valid_against_links_schema({ links: presented_item.links }, "worldwide_organisation")
```